kube/patch-operated.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 set -u
 
-if [  $# -le 1 ]; then
+if [  $# -le 0 ]; then
```

Comment:
Nit, do you mind changing this to `-lt 1`? It's a bit easier on my brain :)

Reply:
it takes two parameters. Only uses first parameter It has to be the prometheus crd name which can be obtained from
kubectl get prometheus -n monitoring

Reply:
Right, `lt 1` and `-le 0` are equivalent. You wrote `-le 0`, which is correct, and I'm suggesting that `-lt 1` is slightly easier to understand. It'd be nice if you can just make that edit, because I can't modify your PR.

Reply:
yes of course :))